android/java/org/chromium/chrome/browser/app/domain/UserAssetModel.java

```diff
@@ -63,34 +62,20 @@ public void fetchAssets(boolean nftsOnly, NetworkInfo selectedNetwork) {
-                                        TokenUtils.TokenType.ALL, userAssets -> {
-                                            TokenUtils.getAllTokensFiltered(mBraveWalletService,
-                                                    mBlockchainRegistry, mSelectedNetwork,
-                                                    mSelectedNetwork.coin,
-                                                    nftsOnly ? TokenUtils.TokenType.NFTS
-                                                             : TokenUtils.TokenType.NON_NFTS,
-                                                    tokens -> {
-                                                        _mAssetsResult.postValue(new AssetsResult(
-                                                                Arrays.asList(tokens),
-                                                                Arrays.asList(userAssets)));
-                                                    });
-                                        });
+                                        TokenUtils.TokenType.ALL,
+                                        userAssets
+                                        -> TokenUtils.getAllTokensFiltered(mBraveWalletService,
+                                                mBlockchainRegistry, mSelectedNetwork,
+                                                nftsOnly ? TokenUtils.TokenType.NFTS
+                                                         : TokenUtils.TokenType.NON_NFTS,
+                                                tokens -> {
+                                                    _mAssetsResult.postValue(
+                                                            new AssetsResult(Arrays.asList(tokens),
+                                                                    Arrays.asList(userAssets)));
+                                                }));
                             }
-                        } else if (mType == WalletCoinAdapter.AdapterType.SEND_ASSETS_LIST) {
-                            assert mSelectedNetwork != null;
-                            TokenUtils.getUserAssetsFiltered(mBraveWalletService, mSelectedNetwork,
-                                    mSelectedNetwork.coin, TokenUtils.TokenType.ALL, tokens -> {
-                                        _mAssetsResult.postValue(new AssetsResult(
-                                                Arrays.asList(tokens), Collections.emptyList()));
-                                    });
-                        } else if (mType == WalletCoinAdapter.AdapterType.BUY_ASSETS_LIST) {
-                            TokenUtils.getBuyTokensFiltered(mBlockchainRegistry, mSelectedNetwork,
-                                    TokenUtils.TokenType.ALL, tokens -> {
-                                        _mAssetsResult.postValue(new AssetsResult(
-                                                Arrays.asList(tokens), Collections.emptyList()));
-                                    });
```

Comment:
`SEND_ASSETS_LIST` and `BUY_ASSETS_LIST` can be safely removed as they were never accessed: these parts now rely on WebUIs.